components/map-menu/components/sections/search/component.jsx

```diff
@@ -11,45 +11,64 @@ import UTMCoords from './components/utm-coords';
 import './styles.scss';
 
 class MapMenuSearch extends PureComponent {
+  menu = [
+    { label: 'Locations', value: 'location' },
+    { label: 'Datasets', value: 'dataset' },
+    { label: 'Coordinates', value: 'coords' },
+    { label: 'Decimal degrees', value: 'decimals' },
+    { label: 'UTM coordinates', value: 'utm' }
+  ]
+
+  searchCategoryMenu() {
+    const { menu } = this;
+    const { isDesktop, searchType: currentSearchType, setMenuSettings } = this.props;
+
+    // Hide menu if mobile and we have a search type active
+    if (!isDesktop && currentSearchType.length > 0) {
+      return null;
+    }
+
+    // Mobile does not care about active, as its not shown on interaction
+    // Desktop always needs a active value, if none present (on resize) show the first menu item as active
+    const currentValue = isDesktop && currentSearchType.length === 0 ? menu[0].value : currentSearchType;
+
+    return (
+      <VerticalMenu
+        className="search-menu"
+        value={currentValue}
+        menu={menu}
+        onClick={value => setMenuSettings({ searchType: value })}
+      />
+    )
+  }
+
   render() {
-    const { searchType, setMenuSettings, isDesktop } = this.props;
+    const { searchType, isDesktop } = this.props;
+
+    // Mobile view displays eater menu or container depending on current action
```

Comment:
spelling mistake? `eater` -> `either` ?

Reply:
> There is a whole other issue of how to access this menu on mobile but we can address that is another PR.

yeah saw that, though it was a feature :D guess we can add it back to the menu in another pr!